<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1443</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>721</width>
      <height>661</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Fabrika Komutu Gönder</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_Connect">
      <property name="geometry">
       <rect>
        <x>572</x>
        <y>30</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Bağlan</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_Port">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>30</y>
        <width>191</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_Baudrate">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>30</y>
        <width>191</width>
        <height>28</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>115200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>57600</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>38400</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>19200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9600</string>
       </property>
      </item>
     </widget>
     <widget class="QTextEdit" name="textEdit_CommandArray">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>80</y>
        <width>391</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Send">
      <property name="geometry">
       <rect>
        <x>572</x>
        <y>80</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Gönder</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_Comport">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>70</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Comport</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_FactoryCommand">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Fabrika komutu</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_LogScreen">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Log ekranı</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_SaveScreenLog">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>130</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Kaydet..</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_ClearScreen">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>130</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Temizle</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_PortReadWrite">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>671</width>
        <height>421</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget_CommandList">
    <property name="geometry">
     <rect>
      <x>722</x>
      <y>0</y>
      <width>721</width>
      <height>661</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Fabrika Komut Listesi</string>
     </attribute>
     <widget class="QListView" name="listView_CommandList">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>211</width>
        <height>591</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_CommandInfo">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>20</y>
        <width>471</width>
        <height>591</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1443</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
